### PR TITLE
changes to metrics

### PR DESCRIPTION
**Pull request by emiliolehoucq**
Addressing #357 

So sorry for the delay, @merveenoyan ! I've been crazy busy. I just suggested some edits to mean reward, which was already in data.ts and added two other metrics.

Two comments:

I noticed @RamAnanth had already contributed the D4RL dataset. I could also add [RL Unplugged](https://github.com/deepmind/deepmind-research/tree/master/rl_unplugged), [Robosuite](https://www.tensorflow.org/datasets/catalog/robosuite_panda_pick_place_can), [MuJoCo locomotion](https://www.tensorflow.org/datasets/catalog/locomotion), [MT Opt dataset](https://www.tensorflow.org/datasets/catalog/mt_opt), and [Mine RL Dataset](https://minerl.io/dataset/), but they don't seem to be hosted by Hugging Face.

Also, there's already three libraries in hub-docs/tasks/src/const.ts I could add [PyTorch](https://huggingface.co/edbeeching/decision-transformer-gym-hopper-medium) and [TensorBoard](https://huggingface.co/ThomasSimonini/MLAgents-Pyramids2), but neither is included in ModelLibrary on hub-docs/js/src/lib/interfaces/Libraries.ts

### CODE REVIEW

**Comment by emiliolehoucq** [comment]
Awesome, it was fun doing this @merveenoyan ! :)

**Comment by merveenoyan** [comment]
@emiliolehoucq can you give me your HF username so I can put credits? 🙂
